test/remote/gateways/remote_sum_up_test.rb

```diff
@@ -126,9 +126,9 @@ def test_failed_void_invalid_checkout_id
   # In Sum Up the account can only return checkout/purchase in pending or success status,
   # to obtain a successful refund we will need an account that returns the checkout/purchase in successful status
   #
-  # For this example configure in the fixtures => :sum_up_account_for_successful_purchases
+  # For this example configure in the fixtures => :sum_up_successful_purchase
```

Comment:
💭  Opinion:
I'm not sure if this comment is required (the fixtures method seems to be enough to inform that the sum_up_successful_purchase needs to be configured).

If you consider conserving it, maybe `test` instead of `example` could be better.